colorcoder/ColorPair.java

```diff
@@ -0,0 +1,37 @@
+package colorcoder;
+
+public class ColorPair {
+    private final MajorColor majorColor;
+    private final MinorColor minorColor;
+    private static final int numberOfMinorColors = MinorColor.values().length;
+
+    public ColorPair(MajorColor major, MinorColor minor)
+    {
+        majorColor = major;
+        minorColor = minor;
+    }
+    public MajorColor getMajor() {
+        return majorColor;
+    }
+    public MinorColor getMinor() {
+        return minorColor;
+    }
+    String ToString() {
+        String colorPairStr = majorColor.getString();
+        colorPairStr += " ";
+        colorPairStr += minorColor.getString();
+        return colorPairStr;
+    }
+    static ColorPair GetColorFromPairNumber(int pairNumber) {
```

Comment:
Encapsulate Get*********** methods in to Util Class

Reply:
moved to utility class